internal/cli/login.go

```diff
@@ -285,20 +283,18 @@ func RunLoginAsMachine(ctx context.Context, inputs LoginInputs, cli *cli, cmd *c
 	if err := keyring.StoreAccessToken(inputs.Domain, token.AccessToken); err != nil {
 		// In case we don't have a keyring, we want the
 		// access token to be saved in the config file.
-		t.AccessToken = token.AccessToken
+		tenant.AccessToken = token.AccessToken
 	}
 
-	if err = cli.addTenant(t); err != nil {
+	if err = cli.Config.AddTenant(tenant); err != nil {
 		return fmt.Errorf("unexpected error when attempting to save tenant data: %w", err)
 	}
 
 	cli.renderer.Newline()
 	cli.renderer.Infof("Successfully logged in.")
 	cli.renderer.Infof("Tenant: %s", inputs.Domain)
 
-	if err := checkInstallID(cli); err != nil {
```

Comment:
Should we remove `checkInstallID()` as it's no longer in use (the only calls were in `RunLoginAsMachine()` and `RunLoginAsUser()`)?